# TEST FILE

# Initial standard deviations
pitch_roll_std: [0.1, 0.1]

# Camera calibration
intrinsics: [640, 480, 320 , 240]
T_cam_imu:
 - [0.41802349, -0.63293951,  0.6516471,  -0.056874]
 - [0.75952742, -0.15001891, -0.63293951,  0.35648 ]
 - [0.49837182,  0.75952742,  0.41802349, -0.84165 ]
 - [0.0,         0.0,         0.0,         1.0     ]

# IMU noise statistics
accelerometer_noise_density: 0.01   
accelerometer_random_walk:   0.001
gyroscope_noise_density: 0.01
gyroscope_random_walk:   0.001

# State options
enable_camera_extrinsic_calibration: true
enable_camera_intrinsic_calibration: true
gravity: 9.81
num_clones: 10
num_persistent_features: 50

# Other options
persistent_feature_init_delay: 1.0
state_transition_order: 1

# Logger level [0: Full, 1: INFO, 2: WARN, 3: ERR, 4: INACTIVE]
# logger_level: 0
logger_level: 4
